src/test/java/automation/pages/BrowsePage.java

```diff
@@ -0,0 +1,20 @@
+package automation.pages;
```

Comment:
What is 'Browse Page'? PageObject must have a name with a business value, e.g. 'GroupManagementPage' or 'MainPage'.

Reply:
there are 3 ways of managing groups in 'GroupManagementPage' and one of then leads through Browse button - thats why I'm using this name, but maybe 'BrowsePanel' will be better? what do you think?

Reply:
If there are three ways to manage groups, then you have to have 3 page objects.

Reply:
Understood